src/tests/system/tests/test_ipa.py

```diff
@@ -596,3 +596,40 @@ def test_ipa__idview_fails_to_apply_on_ipa_master(ipa: IPA):
     assert (
         "ID View cannot be applied to IPA master" in result.stdout
     ), "Did not get an error message when trying to apply ID view on server!"
+
+
+@pytest.mark.topology(KnownTopology.IPA)
```

Comment:
Add an importance level, 'critical', 'high' will run in PRCI, anything lower will not run. IMO, it should be 'high' or 'critical'.